build/docker/Dockerfile

```
@@ -0,0 +1,37 @@
+FROM python:3.11 as base
+
+# set various metadata, loosely following biocontainers standards from:
+# https://biocontainers-edu.readthedocs.io/en/latest/what_is_biocontainers.html
+LABEL base_image="python:3.11"
+LABEL software="pycytominer"
+LABEL about.summary="Python package for processing image-based profiling data"
+LABEL about.home="https://github.com/cytomining/pycytominer"
+LABEL about.documentation="https://pycytominer.readthedocs.io/en/stable/"
+LABEL about.license_file="https://github.com/cytomining/pycytominer/blob/main/LICENSE.md"
+LABEL about.license="SPDX:BSD-3-Clause"
+
+# set the workdir to /app
+WORKDIR /app
+
+# configures poetry to install dependencies without virtualenvs
+# see: https://python-poetry.org/docs/configuration/#virtualenvscreate
+ENV POETRY_VIRTUALENVS_CREATE=false
+
+# install poetry and poetry dynamic versioning
+# hadolint ignore=DL3013
+RUN pip install --no-cache-dir poetry
+
+# create stage for production build (no dev or non-essential deps)
+FROM base as production
+
+# copy the pycytominer source into image
+COPY . .
+
+# install pycytominer using poetry root dependencies and all extras
+RUN poetry install --only-root --all-extras -v --no-interaction
```

```suggestion
RUN poetry install --all-extras -v --no-interaction
```
I don't think we need this `--only-root`

---

Thanks, great point, removing!